test/unit/lib/offline-api/validation/payload-validation-display-field.spec.ts

```diff
@@ -66,7 +66,7 @@ describe('payload validation (display field)', function () {
         [
           {
             type: 'InvalidAction',
-            message: 'Cannot delete field "dessert" on content type "lunch" because it is set as the display field'
```

Comment:
It seems like you changed the intent of this test, and somehow changed the behavior.
The intent here was to ensure that people cannot delete a field that is a `displayField`.  
That probably should still be the case after this.